src/main/java/network/radicle/tools/github/utils/Markdown.java

```diff
@@ -61,8 +80,52 @@ public static String escape(String markdown) {
         return Strings.nullToEmpty(markdown).replace(":", "\\:");
     }
 
+    public static List<MarkdownLink> extractUrls(String markdown) {
```

Comment:
Ideally this class would get refactored to a markdown service

Reply:
@JChrist Fixed. Could you review the changes please?